src/components/dapp-staking/my-staking/styles/dynamic-ads-area.scss

```diff
@@ -0,0 +1,109 @@
+@import 'src/css/quasar.variables.scss';
+
+.wrapper {
+  display: block;
+  margin-top: 48px;
+  margin-bottom: 48px;
+  @media (min-width: $xl) {
+    display: flex;
+    gap: 16px;
+  }
+}
+.wrapper-item {
+  flex: 1 1 0px;
+  height: 211px;
+}
+
+.news-area {
+  margin-bottom: 40px;
+
+  @media (min-width: $md) {
+    margin-bottom: 0px;
+  }
+}
+
+.wrapper--banners {
+  display: flex;
+  gap: 16px;
+  margin-top: 20px;
+  @media (max-width: $md) {
```

Comment:
Could you use `min-width` instead?

Reply:
Fixed [here](https://github.com/AstarNetwork/astar-apps/pull/675/commits/4cc027edeaba605b81883a4487c123762ccc754b) . it can be simply removed